db/migrate/20230224194513_remove_title_from_reviews.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTitleFromReviews < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :reviews, :title
+  end
+end
```

Comment:
You removed these columns from the CreateReviews migration file so you don't need this.
Delete this file then do rails `db:drop` rails `db:create` and rails `db:migrate` and it should work.